include/fi.h

```diff
@@ -144,38 +145,69 @@ static inline uint64_t roundup_power_of_two(uint64_t n)
 #endif /* PT_LOCK_SPIN */
 
 
+#if ENABLE_DEBUG
+#define ATOMIC_IS_INITIALIZED(atomic) assert(atomic->is_initialized)
+#else
+#define ATOMIC_IS_INITIALIZED(atomic)
+#endif
+
```

Comment:
That `#if` is there because I only conditionally compiled in the `is_initialized` field.